### PR TITLE
Use proper svgs for connections and use standard chip component

### PR DESCRIPTION
**Pull request by sudhanshutech**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by MUzairS15 — CHANGES_REQUESTED** [review]
https://github.com/meshery/meshery/assets/79860504/a2ea632a-3b3e-46b4-8627-71b596ee8be7

The adapter synthetic check doesn't work.

File: ui/components/MeshConfigComponent.js [hunk 1]
```diff
@@ -436,22 +435,21 @@ function MesherySettingsNew({ classes, updateProgress, operatorState, k8sconfig
...
-                }
-                icon={<img src="/static/img/kubernetes.svg" className={classes.icon} />}
-                variant="outlined"
-                data-cy="chipContextName"
-              />
-            </Tooltip>
+                )
+              }
+              handlePing={() =>
+                handleKubernetesClick(data[tableMeta.rowIndex].connection_id, tableMeta.rowIndex)
```
Comment (MUzairS15) [inline ui/components/MeshConfigComponent.js:449]: https://github.com/meshery/meshery/pull/9388/files#diff-9514a9ef762e5d87704d59bd30ae5c0a1744322ddcc1bda1a51034bacf4ff690R260 Delete this func, and use the k8s hook.

File: ui/components/MeshConfigComponent.js [hunk 2]
```diff
@@ -634,25 +632,17 @@ function MesherySettingsNew({ classes, updateProgress, operatorState, k8sconfig
...
-                                      src="/static/img/kubernetes.svg"
-                                      className={classes.icon}
-                                    />
-                                  }
-                                  variant="outlined"
-                                  data-cy="chipContextName"
-                                />
-                              </Tooltip>
+                              <ConnectionChip
+                                tooltip={`Server: ${contexts[rowMetaData.rowIndex].server}`}
```
Comment (MUzairS15) [inline ui/components/MeshConfigComponent.js:636]: Make the server URL clickable if not already.

File: ui/components/MeshConfigComponent.js [hunk 3]
```diff
@@ -744,54 +722,32 @@ function MesherySettingsNew({ classes, updateProgress, operatorState, k8sconfig
...
-                                        <img
-                                          src="/static/img/nats-icon-color.svg"
-                                          className={classes.icon}
-                                        />
-                                      }
-                                      variant="outlined"
-                                      data-cy="chipNATS"
-                                    />
-                                  </Tooltip>
+                                    label={'NATS'}
```
Comment (MUzairS15) [inline ui/components/MeshConfigComponent.js:746]:
```suggestion
                                    label={'Meshery Broker'}
```

Comment (MUzairS15) [reply ui/components/MeshConfigComponent.js:746]: https://github.com/meshery/meshery/pull/9388/files#diff-9514a9ef762e5d87704d59bd30ae5c0a1744322ddcc1bda1a51034bacf4ff690R795
Same thing here.

Comment (MUzairS15) [reply ui/components/MeshConfigComponent.js:746]: And here as well.
https://github.com/meshery/meshery/pull/9388/files#diff-9514a9ef762e5d87704d59bd30ae5c0a1744322ddcc1bda1a51034bacf4ff690R800

**Review by MUzairS15 — CHANGES_REQUESTED** [review]

**Comment by sudhanshutech** [comment]
ok i will update the changes, i will be travelling to college for exams so i will try to fix asap

**Comment by leecalcote** [comment]
Thanks, @sudhanshutech. Heads-up: merge conflict.

**Comment by leecalcote** [comment]
Thanks, @sudhanshutech. Heads-up: merge conflict.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Chips still don't invoke an ad hoc connectivity test on click, @sudhanshutech

**Comment by leecalcote** [comment]
Let me know when you have actually fixed this and it is ready for re-testing, @sudhanshutech

**Comment by leecalcote** [comment]
> Let me know when you have actually fixed this and it is ready for re-testing, @sudhanshutech

@sudhanshutech, ping

**Comment by leecalcote** [comment]
Merge conflicts.

**Comment by sudhanshutech** [comment]
Its already been done, i guess only metrics page need a update that i will add. We will use the sistent chip so this not required now.

**Pull request closed without merging**